src/lib/group.test.ts

```diff
@@ -10,6 +10,10 @@ import {
 } from 'snarkyjs';
 
 describe('group', () => {
+  let g = Group({
+    x: -1,
+    y: 2,
+  });
```

Comment:
love it!! I didn't know there was a neat example group element. we should make this the generator